UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.ts

```diff
@@ -549,8 +550,31 @@ export class MangaReaderComponent implements OnInit, AfterViewInit, OnDestroy {
         takeUntilDestroyed(this.destroyRef)
       ).subscribe(() => {});
 
+      this.generalSettingsForm.get('widthSlider')?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
+        //update the css width
+        const styleSheets = document.styleSheets;
+        for(let i = 0; i < styleSheets.length; i++){
+          const styleSheet = styleSheets[i];
+          const rules = styleSheet.cssRules;
+          for(let j = 0; j < rules.length; j++){
+            let rule = rules[j] as CSSStyleRule;
+            if(!rule ||
+              !(rule.cssText.includes("img[_ngcontent-"))) continue;
+            rule.style.width = val + "%";
+          }
+        }
+      });
 
-
+      this.generalSettingsForm.get('fittingOption')?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
+          let slider = document.getElementById("page-fitting-slider") as HTMLInputElement;
```

Comment:
You don't need to do this type of thing in Angular. We should enable/disable only. We shouldn't hide it from the UI. 

Also keep the subscribe blocks empty. Put this in a tap() so that subscribe can be empty.